src/axom/sidre/docs/sphinx/mfem_sidre_datacollection.rst

```diff
@@ -0,0 +1,74 @@
+.. ## Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _mfem-sidre-datacollection:
+
+******************************************************
+Using Sidre with MFEM
+******************************************************
+
+The ``MFEMSidreDataCollection`` class implements `MFEM <https://mfem.org>`_'s 
+``DataCollection`` interface for collecting data for a given simulation.
+It combines fields and the mesh on which they are defined.  
+
+``MFEMSidreDataCollection`` internally uses the Mesh Blueprint, a standardized
+ representation, to store its data.
+
+See the :ref:`Conduit page <sidre-conduit>` for more information on the Mesh Blueprint.
+
+Uses
+--------------
+
+The data in an instance of the ``MFEMSidreDataCollection`` class can be saved to a file using a variety of formats.  
+This provides visualization and restart capabilities, as these files can also be
+loaded back into an instance of the class.
+
+Current options for output formats (protocols) include:
+
+   - ``sidre_hdf5``
+   - ``sidre_conduit_json``
+   - ``sidre_json``
+   - ``conduit_hdf5``
+
+.. Note::
+   The ``AXOM_USE_HDF5`` build option must be enabled to use the HDF5-based formats.
+
+A typical visualization example might look like the following:
+
+.. code-block::
+
+   mfem::Mesh mesh;
```

Comment:
Would it be possible to move this code snippet to a `sidre` example, that way we are sure that it continues to work?